tornado/test/runtests.py

```diff
@@ -28,6 +28,7 @@
     'tornado.test.condition_test',
     'tornado.test.curl_httpclient_test',
     'tornado.test.escape_test',
+    'tornado.test.event_test',
```

Comment:
I generally prefer to keep the test file structure parallel to the source files, so combine condition_test and event_test into locks_test.py.